Isort no longer uses futures in its codebase. We removed it from Solaris,
and hence we have to remove it from here as well. This patch can be removed
once we get rid of Python 2.7.

Upstream most likely already removed this dependency.

--- isort-4.3.21/isort.egg-info/requires.txt
+++ isort-4.3.21/isort.egg-info/requires.txt
@@ -1,6 +1,5 @@
 
 [:python_version < "3.2"]
-futures
 backports.functools_lru_cache
 
 [pipfile]
--- isort-4.3.21/setup.py
+++ isort-4.3.21/setup.py
@@ -28,7 +28,6 @@ setup(name='isort',
           'xdg_home': ['appdirs>=1.4.0'],
       },
       install_requires=[
-          'futures; python_version < "3.2"',
           'backports.functools_lru_cache; python_version < "3.2"',
       ],
       python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
